src/screens/OtherProfile/index.js

```diff
@@ -135,9 +140,6 @@ const OtherProfile = () => {
         dataFeed: feedOtherProfile,
         dataCache: cacheFeed
       });
-      if (Array.isArray(result.data) && result.data.length === 0) {
-        setIsLastPage(true);
-      }
       if (offset === 0) {
         setOtherProfileFeed(mapNewData, dispatchOtherProfile);
         StorageUtils.otherProfileFeed.setForKey(otherId, JSON.stringify(mapNewData));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for an empty result array has been removed in the new code. This check was previously used to set `isLastPage` to `true`, indicating that there were no more feeds to fetch. With the removal of `isLastPage`, it's important to ensure that the new code handles the case where there are no more feeds to fetch. If not handled correctly, this could lead to unnecessary network requests or errors.

Please ensure that the new code correctly handles the case where there are no more feeds to fetch.



<!-- This is an auto-generated comment by OSS CodeRabbit -->